### PR TITLE
Added AQM datamodel

### PR DESCRIPTION
**Pull request by shyam28598**

### CODE REVIEW

File: AirQualityMonitoring/schema.json [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+                },
+                "airQualityLevel": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. Air Quality Category Indication. Qualitative level defined according to the local health agencies. For example, 'GOOD', 'MODERATE', 'POOR', 'UNHEALTHY', 'SEVERE', 'HAZARDOUS' etc."
+                },
+                "aqiMajorPollutant": {
+                    "type": "string",
+                    "description": "Property. Model:'https://schema.org/Text'. Major pollutant in the Air Quality Index (AQI)."
+                },
+                "": {
```
Comment (albertoabellagarcia) [inline AirQualityMonitoring/schema.json:47]: The name of the property?

File: AirQualityMonitoring/schema.json [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+                    "type": "object",
+                    "description": "Property. ",
+                    "properties": {}
```
Comment (albertoabellagarcia) [inline AirQualityMonitoring/schema.json:50]: Object's properties should be defined. there are approaches for generic needs. I'd mail you if necessary.

File: AirQualityMonitoring/schema.json [hunk 3]
```diff
@@ -0,0 +1,166 @@
...
+                },
+                "co2": {
+                    "type": "object",
+                    "description": "Property. Observed value of environmental pollutant Carbon Dioxide (CO2). Value is an object containing attributes representing statistical aggregates derived from past data.",
+                    "properties": {}
```
Comment (albertoabellagarcia) [inline AirQualityMonitoring/schema.json:55]: See first this https://smartdatamodels.org/co2
Why object?

File: AirQualityMonitoring/schema.json [hunk 4]
```diff
@@ -0,0 +1,166 @@
...
+                },
+                "co": {
+                    "type": "object",
+                    "description": "Property. Observed value of environmental pollutant Carbon Monoxide (CO). Value is an object containing attributes representing statistical aggregates derived from past data.",
+                    "properties": {}
```
Comment (albertoabellagarcia) [inline AirQualityMonitoring/schema.json:60]: Why object
See https://smartdatamodels.org/co

File: AirQualityMonitoring/schema.json [hunk 5]
```diff
@@ -0,0 +1,166 @@
...
+                },
+                "relativeHumidity": {
```
Comment (albertoabellagarcia) [inline AirQualityMonitoring/schema.json:62]: shouldn't be this a number? 
https://smartdatamodels.org/relativeHumidity

File: AirQualityMonitoring/schema.json [hunk 6]
```diff
@@ -0,0 +1,166 @@
...
+                    "$ref": "http://geojson.org/schema/Geometry.json"
+                },
+                "illuminance": {
+                    "type": "string",
+                    "format": "uri",
+                    "description": "Relationship. Model:'https://schema.org/URL'. Observed instantaneous ambient light intensity."
```
Comment (albertoabellagarcia) [inline AirQualityMonitoring/schema.json:68]: A uri? shouldn't be a number? https://smartdatamodels.org/illuminance

File: AirQualityMonitoring/schema.json [hunk 7]
```diff
@@ -0,0 +1,166 @@
...
+                },
+                "no2": {
+                    "type": "object",
+                    "description": "Property. Observed value of environmental pollutant Nitrogen Dioxide (NO2). Value is an object containing attributes representing statistical aggregates derived from past data.",
+                    "properties": {}
```
Comment (albertoabellagarcia) [inline AirQualityMonitoring/schema.json:73]: same as previous

File: AirQualityMonitoring/ADOPTERS.yaml [hunk 8]
```diff
@@ -0,0 +1,11 @@
+
```
Comment (albertoabellagarcia) [inline AirQualityMonitoring/ADOPTERS.yaml:1]: it would be good to mention where the data model is used

File: AirQualityMonitoring/schema.json [hunk 9]
```diff
@@ -0,0 +1,166 @@
+{
```
Comment (albertoabellagarcia) [inline AirQualityMonitoring/schema.json:1]: I have reviewed first detail by detail and then globally. My question, how could we ensure a quick and simple interoperability if we do not define what are the objects' properties? There is not a problem with defining them  as objects and defining all the possible values (or extending later). But how could you prepare on the other end the reading of the values? In fact, what you are doing is creating a not documented standard (under the need for being very flexible). 
Let's comment this

File: AirQualityMonitoring/examples/example.jsonld [hunk 10]
```diff
@@ -0,0 +1,102 @@
+{
```
Comment (albertoabellagarcia) [inline AirQualityMonitoring/examples/example.jsonld:1]: This payload is not validated by the schema.json

**Review by albertoabellagarcia — COMMENTED** [review]
Thank you very much for the contribution

Please attend the comments and do not hesitate to contact us for further explanation or to attend to the "Open session" on Mondays.
https://smartdatamodels.org/index.php/summary-open-session/

Thanks

File: AirQualityMonitoring/schema.json (hunk 9 above)
Comment (shyam28598) [reply AirQualityMonitoring/schema.json:1]: Hey Alberto, I have made some changes with respect to the schema.json document I believe it should have all the properties required now. If there are modifications required in the payload files please let me know I will modify them as well.

Comment (shyam28598) [reply AirQualityMonitoring/schema.json:1]: With regard to the reading of the values we have a different file called the Data Descriptor in which the detailed description of each and every property is mentioned.You can check out Example 1 in this [IUDX Vocabulary](https://voc.iudx.org.in/EnvAQM). This contains the detailed information  about each of the properties in the data model.

**Comment by shyam28598** [comment]
> right

Hey Alberto, I have taken a look at the incubated data model and made some changes accordingly(Please check modified-schema.json), since some properties remain as [timeseriesaggregation](https://voc.iudx.org.in/TimeSeriesAggregation


) it will contain max,min,avg and instvalue and the pollutants have been put in an array as mentioned in the incubated schema, Please get back to me if there are any more changes

**Review by albertoabellagarcia — APPROVED** [review]
some additional changes has been introduced in the official version

**Comment by albertoabellagarcia** [comment]
reviewed before publishing

**Comment by albertoabellagarcia** [comment]
see published one
https://smartdatamodels.org/index.php/airqualitymonitoring/

**Pull request merged**